_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+
+### 3: The Vareti'het
+
+<img class="centered" src="/images/blog/ka-het/vareti-het-exploded.jpg" width="570" height="370" />
+
+The capital ship every fleet needs, an incredible force with impenetrable shields, reactive armor and a massive electric disruptor, nullifying in a few shots even the strongest generators. Its troubled development, caused from the unprecedented size of the super-sized organism and of the shield generators placed on top of it made it to end up with a somewhat different design from its predecessors compared to the original concept of an upsized Fetri; its capabilities have only benefited from this, however. Alone, this ship is formidable; in a fleet, it's unstoppable.
+After leaving the main Ka'het station, much like the Telis, the Vareti start listening for orders and begin patrolling the entire cluster; this way they will be ready to intervene as soon as they're called into action, without the undesirable delay given by the hyperlane system.
+
+## Appendix: unique capabilities of the Ka'sei and Ka'het
+
+<img class="centered shadowed" src="/images/blog/ka-het/fetri-sei-replication.jpg" width="480" height="320" />
+
+> ***The four stages of Ka'sei replication***
+
+The self-replication capabilities of the Ka'sei start with the selection of a suitable asteroid to use for it; it will provide the ship with most, if not all, of the necessary materials for this activity and act as a base for the Fetri for the years to come. Once this has been done, and as long as the Ka'sei has sufficient resources, it will open the aft workstation to begin growing an organism completely identical to itself, moving only to gather additional materials as needed. In around 40 months the new organism becomes self-sufficient and can separate, first remaining connected to the original's life support through additional umbilicals and then surviving completely on its own; in fact, during this phase the older Fetri will use the forward workstation to construct the hull and mechanical systems of the new organism, a phase which takes between 5 and 7 weeks depending on material availability. Two identical Ka'sei now exist, both equally as capable and both ready to continue their mission.
+There was much internal debate during the development of the Ka'het about these systems, and whether they would be harmful or beneficial to the overall architecture; the exponential growth was considered by some to be essential to success of the fleet, while others found its size and added requirements unacceptable. In the end, the Assured Continued Production doctrine meant that the benefits were seen as irrelevant for the intended use, given that if a large part of the fleet was lost in a single event building it back would have been impossible.
```

Comment:
```suggestion
The self-replication capabilities of the Ka'sei start with the selection of a suitable asteroid to use for it; it will provide the ship with most, if not all, of the necessary materials for this activity and act as a base for the Fetri for the years to come. Once this has been done, and as long as the Ka'sei has sufficient resources, it will open the aft workstation to begin growing an organism completely identical to itself, moving only to gather additional materials as needed. In around 40 months the new organism becomes self-sufficient and can separate, first remaining connected to the original's life support through additional umbilicals and then surviving completely on its own; in fact, during this phase the older Fetri will use the forward workstation to construct the hull and mechanical systems of the new organism, a phase which takes between 5 and 7 weeks depending on material availability. Two identical Ka'sei now exist, both equally as capable and ready to continue their mission.
There was much internal debate during the development of the Ka'het about these systems, and whether they would be harmful or beneficial to the overall architecture; the exponential growth was considered by some to be essential to success of the fleet, while others found its size and added requirements unacceptable. In the end, the Assured Continued Production doctrine meant that the benefits were seen as irrelevant for the intended use, given that if a large part of the fleet was lost in a single event building it back would have been impossible.
```